### PR TITLE
chore(*): add install docs and scripts

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add install docs and scripts

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by simon-cj** [comment]
@lqxu @hewenguang @FogDong

File: docs/docs_zh/installation.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+安装 klever-web 之前，需安装依赖的如下组件：
+* [istio](https://github.com/istio/istio)
+* [seldon-core](https://github.com/SeldonIO/seldon-core)
+* [harbor](https://github.com/goharbor/harbor)
+* [klever-model-registry](https://github.com/kleveross/klever-model-registry)
+
+## 安装 klever-web
+```bash
+$ git clone https://github.com/kleveross/web
```
Comment (hewenguang) [inline docs/docs_zh/installation.md]: git clone https://github.com/kleveross/klever-web

File: docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+To use klever-web, you MUST install some dependent components as follows:
+* [istio](https://github.com/istio/istio)
+* [seldon-core](https://github.com/SeldonIO/seldon-core)
+* [harbor](https://github.com/goharbor/harbor)
+* [klever-model-registry](https://github.com/kleveross/klever-model-registry)
+
+## Install klever-web
+```bash
+$ git clone https://github.com/kleveross/klever-web
```
Comment (hewenguang) [inline docs/installation.md:11]: 这里需要安装前端依赖吧？

**Review by grootfish — COMMENTED** [review]
/lgtm

File: docs/installation.md (hunk 2 above)
Comment (FogDong) [reply docs/installation.md:11]: 这里只是 helm 安装

**Review by FogDong — COMMENTED** [review]
/lgtm
/hold cancel

**Pull request merged**